components/FormComponents.js

```diff
@@ -2,7 +2,24 @@ import styled from "styled-components";
 import { IconChevronDown, IconChevronUp, IconList, IconPlus } from "./Icons";
 
 import { PTag } from "./HtmlComponents";
-import { useState } from "react";
+import { useEffect, useState } from "react";
+
+const FormMain = styled.main`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
```

Comment:
a main element already has width 100% by default

Reply:
@F-Kirchhoff i'll change that :)